django_cryptography/utils/crypto.py

```diff
@@ -18,30 +18,28 @@ class InvalidToken(Exception):
     pass
 
 
-def salted_hmac(key_salt, value, secret=None):
```

Comment:
This refactor breaks parity with [django.utils.crypto.salted_hmac](https://github.com/django/django/blob/master/django/utils/crypto.py#L29), the goal is to have a "drop in" replacement.